protoc-gen-grpc-gateway/internal/gengateway/generator_test.go

```diff
@@ -77,6 +77,7 @@ func newExampleFileDescriptorWithGoPkg(gp *descriptor.GoPackage, filenamePrefix
 
 func TestGenerator_Generate(t *testing.T) {
 	g := new(generator)
+	g.reg = descriptor.NewRegistry()
```

Comment:
Was this change necessary?

Reply:
Yes. `new(generator)` was not calling `NewRegistry` and without calling `NewRegistry` the map was not getting initialized, resulting in a panic. Let me know if there's a way to fix this without changing the test itself.